serialization/aut/src/main/java/net/automatalib/serialization/aut/InternalAUTParser.java

```diff
@@ -187,13 +191,20 @@ private int parseNumberAndShift() {
         final StringBuilder sb = new StringBuilder();
 
         char sym = currentLineContent[currentPos];
+        if(sym == '-') {
+            throw new IllegalArgumentException(buildErrorMessage("Negative numbers not compatible with AUT format"));
+        }
```

Comment:
added condition to throw more informative exception upon encountering negative numbers